Code/Comscore/Source/integration/ComscoreAdapter.swift

```diff
@@ -444,10 +446,9 @@ class THEOComScoreAdapter: NSObject {
         if configuration.debug { print("[THEOplayerConnectorComscore] AD_BEGIN event") }
         if let ad = event.ad {
             currentAdDuration = 0
-            if let duration = player.duration {
-                if duration != .infinity && duration != .nan {
-                    currentAdDuration = Int(duration * 1000)
-                }
+            if let linearAd = ad as? LinearAd,
+               let duration: Int = linearAd.duration {
+                self.currentAdDuration = duration
```

Comment:
Just to make sure: currentAdDuration was previously converted to msec? Is that also msec on the linearad? Or maybe that was a mistake?

Reply:
Good catch! I will convert it to an msec.